### ISSUE TITLE
description Attribute Not Effective When db_constraint=False

### ISSUE THREAD

**Issue by yuanshibtc** [tortoise/tortoise-orm#1679]
When defining a ForeignKeyField in Tortoise ORM, setting the description attribute does not seem to take effect if db_constraint=False is also set. 

```
class Event(Model):
    id = fields.IntField(primary_key=True)
    name = fields.TextField()
    tournament: fields.ForeignKeyRelation[Tournament] = fields.ForeignKeyField(
        "models.Tournament", related_name="events", db_constraint=False, description="Tournament", null=False, index=True
    )
    participants: fields.ManyToManyRelation["Team"] = fields.ManyToManyField(
        "models.Team", related_name="events", through="event_team", db_constraint=False, on_delete=fields.NO_ACTION
    )

    def __str__(self):
        return self.name
```
```
CREATE TABLE `event` (
  `id` int NOT NULL AUTO_INCREMENT,
  `name` longtext NOT NULL,
  `tournament_id` int NOT NULL,
  PRIMARY KEY (`id`),
  KEY `idx_event_tournam_c08428` (`tournament_id`)
) ENGINE=InnoDB AUTO_INCREMENT=3 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
```
<img width="846" alt="image" src="https://github.com/user-attachments/assets/dae800c8-892e-45ee-9f62-01394eb05441">

**Comment by abondar** [comment]
Yeah, when we introduced `db_constraint` we forgot about this case

I fixed it in develop, but not sure when I will release it, as I am not sure that I want to make release just for this one fix

I'll write here when it will be released

**Comment by abondar** [comment]
Released as 0.21.6
